### DISCUSSION TITLE
Elegoo Mars 2 Pro

### DISCUSSION THREAD

**Discussion by Millerboy3** [rudetrooper/Octoprint-Chituboard#41] [General]
By chance does anyone have the older supported fw for the Mars 2 Pro laying around? It seems elegoo has removed the old downloads making them inactive so you can't download them anymore. I would greatly appreciate it! Thanks!

**Comment by ChristophCaina** [comment]
god damn... why why why... :D
I just wanted to start with setting up octoprint for my "freshly updated" mars2pro... :-(
... THEN I found out about the recent issues with the changes elegoo did in their firmware...

MAYBE - I do have a copy of the old stuff from the USB stick... let me check

[EDIT]
Sorry... It seems that I have just deleted the files after the upgrade a few weeks ago :-(
